starter/app/src/main/res/layout/fragment_select_location.xml

```diff
@@ -11,10 +11,28 @@
     </data>
 
     <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/main_container"
         android:layout_width="match_parent"
         android:layout_height="match_parent">
-        <!--TODO: Add the map fragment for the user to select the location-->
```

Comment:
@graffiti75 Kindly, why did you decide to delete this TODO?

Reply:
New PR created: https://github.com/udacity/nd940-android-kotlin-c4-starter/pull/14